libs/shared/ui/src/lib/components/board/board.tsx

```diff
@@ -0,0 +1,476 @@
+import { motion } from 'framer-motion'
+import { type DragEvent, type ReactNode, useId, useState } from 'react'
+
+/**
+ * Inspired by https://www.youtube.com/watch?v=O5lZqqy7VQE
+ * https://www.hover.dev/components/boards
+ * https://codesandbox.io/p/sandbox/fvy4m4
+ *
+ * NOTE: This component doesn't follow the "compound pattern" as
+ * - it requires complex typing
+ * - it needs to pass props to child components like drag n drop handles
+ * - it is rarely used so no big of a need to be highly configurable
+ * However in the future it can be turn into a compound pattern using a react context
+ */
+
+type SetData<T extends ColumnType> = ({
+  sourceColumnId,
+  sourceCardId,
+  targetColumnId,
+  targetCardId,
+  after,
+  newData,
+}: {
+  sourceColumnId: string
+  sourceCardId?: string
+  targetColumnId: string
+  targetCardId?: string
+  after?: boolean
+  newData: T[]
+}) => void
+
+const Board = <T extends ColumnType>({
+  data,
+  setData,
+  showCardIndicator = true,
+  emptyState = null,
+}: {
+  data: T[]
+  setData: SetData<T>
+  showCardIndicator?: boolean
+  emptyState?: ReactNode
+}) => {
+  const instanceId = useId()
+
+  return (
+    <div className="flex h-full w-full overflow-scroll">
```

Comment:
```suggestion
    <div className="flex h-full w-full overflow-scroll pb-5">
```

You need to have an padding because otherwise the scrollbar is displayed in the cards, check the previous version

Reply:
Fixed https://github.com/Qovery/console/pull/1671/commits/e912fe8aed53788c3a3457be1e6e503cf758b8fd